sageintacctsdk/apis/allocation_entry.py

```diff
@@ -19,8 +20,20 @@ def get_all_generator(self, field: str = None, value: str = None, fields: list =
         fields = self.get_lookup()
         if fields and 'Type' in fields and fields['Type'] and 'Relationships' in fields['Type'] and fields['Type']['Relationships'] and 'Relationship' in fields['Type']['Relationships']:
             user_defined_dimensions = fields['Type']['Relationships']['Relationship']
-
+ 
         for allocation_field in user_defined_dimensions:
             allocation_entries_fields.append(allocation_field['RELATEDBY'])
 
-        yield from super().get_all_generator(fields=allocation_entries_fields, field=field, value=value)
+        try:
+            yield from super().get_all_generator(fields=allocation_entries_fields, field=field, value=value)
+        except Exception as e:
+            error_response = e.response
```

Comment:
do everything safely, if response is present, also do WrongParamsError or whatever the exact exception is